openedx/features/partners/tests/test_g2a_registration.py

```diff
@@ -0,0 +1,107 @@
+from ddt import ddt, data
+
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+
+from openedx.features.partners.tests.factories import PartnerFactory, OrganizationFactory
+from openedx.core.lib.api.test_utils import ApiTestCase
+from lms.djangoapps.onboarding.models import Organization
+
+
+@ddt
+class RegistrationViewTest(ApiTestCase):
+    """Tests for the registration"""
+
+    NAME = "bob james"
+    COUNTRY = "Pakistan"
+    ORGANIZATION = "arbisoft"
+    EMAIL = "bob@example.com"
+    USERNAME = "bob"
+    PASSWORD = "Test@12345"
+    TERMS_OF_SERVICE = True
+
+    def setUp(self):
+        super(RegistrationViewTest, self).setUp()
+        self.organization = OrganizationFactory(
+            label='arbisoft'
+        )
+        self.partner = PartnerFactory.create(slug="give2asia", label="arbisoft")
+        self.url = reverse("partner_register", args=["give2asia"])
+
+    def test_put_not_allowed(self):
+        response = self.client.put(self.url)
+        self.assertHttpMethodNotAllowed(response)
+
+    def test_delete_not_allowed(self):
+        response = self.client.delete(self.url)
+        self.assertHttpMethodNotAllowed(response)
+
+    def test_register(self):
```

Comment:
Lets update test method name, method name should tell briefly about test scenario

Reply:
done